Help: fixed errors, typos and some inconsistencies (#1322)
GrandOrgue/grandorgue: CHANGELOG.md

> - Help: fixed errors, did spell check and some cleanup

Reviewer: ```suggestion
- Fixed typos and spelling errors in the Help
```
